src/model/ModelHVAC.i

```diff
@@ -438,6 +444,12 @@ SWIG_MODELOBJECT(SetpointManagerFollowGroundTemperature,1);
       }
     }
 
+    public partial class HeatExchangerDesiccantBalancedFlowPerformanceDataType1 : ModelObject {
```

Comment:
@joseph-robertson here is where I messed up during the partial classes...

Reply:
Should be fixed via https://github.com/NREL/OpenStudio/pull/4317/commits/a2e01038d378b053ffb0bb85c447127802c8be5f